datafusion/sqllogictest/test_files/functions.slt

```diff
@@ -483,7 +483,7 @@ statement error Did you mean 'arrow_typeof'?
 SELECT arrowtypeof(v1) from test;
 
 # Scalar function
-statement error Did you mean 'to_timestamp_seconds'?
```

Comment:
🤔  this looks like there may be special case handling for function hinting that doesn't work for user defined functions. 

I'll file a ticket to add that feature to user defined functions too

Reply:
Fixed in https://github.com/apache/arrow-datafusion/pull/9407